### PR TITLE
Feature/backend/wxs

### PR DESCRIPTION
**Pull request by bestksl**
fix conflicts.

### CODE REVIEW

File: hobbymatcher/src/main/test/java/com/hobbymatcher/test/BaseTest.java [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
- *
- */
-@RunWith(SpringJUnit4ClassRunner.class)
-// 告诉junit spring配置文件的位置
-@ContextConfiguration({ "classpath:spring/spring-dao.xml", "classpath:spring/spring-service.xml"})
-public class BaseTest {
-
-}
\ No newline at end of file
+//package com.hobbymatcher.test;
```
Comment (seige13) [inline hobbymatcher/src/main/test/java/com/hobbymatcher/test/BaseTest.java]: Can we remove this file and others if they are just commented out?

**Comment by seige13** [comment]
Do we need the generated class files or test output files in this pull request?

**Comment by wbwxshao** [comment]
> Do we need the generated class files or test output files in this pull request?
Which files are you talking about specifically?

**Comment by wbwxshao** [comment]
Can we keep it for now?
Let's see if the backend can integrate with frontend

**Comment by seige13** [comment]
If we keep the generated files, it will make reviewing changes more difficult. I will approve this anyways. We should also look into removing the files which are commented out

**Comment by wbwxshao** [comment]
I just update the date to datetime. Do I need to make another pull request or this is fine?

**Review by seige13 — APPROVED** [review]

**Pull request merged**